example/TestApp.re

```diff
@@ -34,6 +34,13 @@ let draw = canvas => {
   Path.lineTo(path, 590., 50.);
   Path.cubicTo(path, -490., 50., 1130., 430., 50., 430.);
   Path.lineTo(path, 590., 430.);
+  Path.addCircle(path, 100., 300., 100., Clockwise);
+
+  let roundRect = Rect.makeLtrb(300., 400., 240., 280.);
+  Path.addRoundRect(path, roundRect, 25., 25., Clockwise);
+  Path.addCircle(path, 100., 300., 100., Clockwise);
+
+  Path.close(path);
```

Comment:
```suggestion
  Path.close(path);
  Path.addCircle(path, 100., 300., 100., Clockwise);

  let roundRect = Rect.makeLtrb(300., 400., 240., 280.);
  Path.addRoundRect(path, roundRect, 25., 25., Clockwise);
  Path.addCircle(path, 100., 300., 100., Clockwise);
```

For me I don't see the close take effect expect I move before other call. Is it a bug only on my computer ? And you have two call with the same coordinates is it intended ? It make the stoke a bit more thick.

Reply:
I think it is ok:
`close() has no effect if SkPath is empty or last SkPath SkPath::Verb is kClose_Verb.` addCircle should add a closed shape.